design-system/src/commonMain/kotlin/components/inputs/LoginInput.kt

```diff
@@ -0,0 +1,114 @@
+package components.inputs
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldColors
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import colors.TextColor
+import dev.icerock.moko.resources.ImageResource
+import dev.icerock.moko.resources.StringResource
+import dev.icerock.moko.resources.compose.painterResource
+import dev.icerock.moko.resources.compose.stringResource
+import instaclone.resources.MR
+import layout.horizontalPadding
+
+@Composable
+fun UsernameInput(
+    currentValue: String,
+    onTextChange: (String) -> Unit
+) {
+    TextField(
+        value = currentValue, onValueChange = {
+            onTextChange(it)
+        },
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = horizontalPadding),
+        placeholder = { Text(text = stringResource(MR.strings.placeholder_user_login)) },
+        colors = getTextFieldColors(),
+        shape = RoundedCornerShape(5.dp),
+    )
+}
+
+@Composable
+fun PasswordInput(
+    currentValue: String,
+    onTextChange: (String) -> Unit
+) {
+    val passwordMask = remember {
+        mutableStateOf<VisualTransformation>(PasswordVisualTransformation())
+    }
+    val passwordIcon = remember {
```

Comment:
I think you can just pass the `MR.strings.password_visible` as an argument, I think we don't need a `remember` here